examples/hello_world.cz

```diff
@@ -0,0 +1,5 @@
+with sys
```

Comment:
Should we need to import sys?

Reply:
I thought that was your idea to use stdin and stdout

Reply:
Yeah, it was, but maybe `sys` could be implicitly imported

Reply:
sure, sounds good